amoro-ams/amoro-ams-server/pom.xml

```diff
@@ -186,6 +186,13 @@
             <artifactId>derby</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.33</version>
```

Comment:
How about declaring the version in the properties section in parent pom.xml?

Reply:
And I am not sure why this dependency is necessary.

Reply:
The purpose of adding this dependency is to make it easier to start AMS in IDEA and debug directly.